model/plugins/get_certificates/operations.smithy

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+use opensearch.openapi#vendorExtensions
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/2.7/security/access-control/api/#get-certificates"
+)
+
+@vendorExtensions(
+    "x-operation-group": "get_certificates",
```

Comment:
remember to put this operation in `security` namepsace. Same with other operations.